notifications_utils/logging.py

```diff
@@ -106,9 +106,8 @@ def get_handlers(app):
 
     if not app.debug:
         # machine readable json to file
-        handler = logging.handlers.TimedRotatingFileHandler(
-            filename='{}-{}.json'.format(app.config['NOTIFY_LOG_PATH'], os.getpid()),
-            when='midnight'
+        handler = logging.handlers.WatchedFileHandler(
+            filename='{}-{}.json'.format(app.config['NOTIFY_LOG_PATH'], os.getpid())
```

Comment:
A bit late to the party, do we need the pid anymore?

Reply:
think we do for FTP while we still have logs from both gunicorn and celery